shared/git_monkey_patches.rb

```diff
@@ -14,16 +14,20 @@ module Git
   # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
   # See the License for the specific language governing permissions and
   # limitations under the License.
-   class Lib
+  class Lib
     # Monkey patch ls_files until https://github.com/ruby-git/ruby-git/pull/320 is resolved
-    def ls_files(location=nil)
-      location ||= '.'
+    def ls_files(location = nil)
+      location ||= "."
       hsh = {}
-      command_lines('ls-files', ['--stage', location]).each do |line|
+      command_lines("ls-files", ["--stage", location]).each do |line|
         (info, file) = line.split("\t")
         (mode, sha, stage) = info.split
+
+        # rubocop:disable Security/Eval
```

Comment:
🙈🙉🙊

Reply:
I didn't check if the actual `git` implementation has this? If if does, it should be fine I think?

Reply:
I didn't check either, but it's *probably* fine